hack/package/package.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -ex
+
+# VERSION the version to give to the package
+VERSION=$1
+
+# The iteration to give to the package. RPM calls this the 'release'.
+# FreeBSD calls it 'PORTREVISION'. Debian calls this 'debian_revision', eg:
+ITERATION=$2
+
+# the gpg keys directory
+KEYDIR=$3
+
```

Comment:
Add an option to choose build deb/rpm package or both. for example:
```
PKGTYPE=${4:-"all"}
```
And change the following build code like:
```
if [[ $PKGTYPE == "all" || $PKGTYPE == "deb" ]] ; then 
   # build deb image
   # build deb package
fi
if [[ $PKGTYPE == "all" || $PKGTYPE == "rpm" ]] ; then 
  #build rpm image
   # build rpm package
fi

```